libr/asm/arch/avr/format.c

```diff
@@ -27,7 +27,9 @@
 #include <string.h>
 #include <stdlib.h>
 #include "format.h"
-
+#ifdef _MSC_VER
+#define strcasecmp stricmp
```

Comment:
yes the problem are with files where no r_types.h/r_util.h inclusion how in this case. The solution is add int r_rtypes and add include to r_type.h in each file with stricmp/strcasecmp